keentools_facebuilder/settings.py

```diff
@@ -69,6 +69,12 @@ def update_pin_size(self, context):
     FBLoader.viewport().update_pin_size()
 
 
+def update_scale(self, context):
```

Comment:
We have to check how to work the change of scale in pin-mode. Most likely you also need to update the mesh